CivicActions/openacr
# Criteria counts as table

Fixing https://github.com/CivicActions/openacr-editor/issues/23 in the CLI first. This took me a while as it was more complicated to count values into a table.

Testing steps:
1. In the code changes you can see the previous summary being replaced with a table. Best way to view the new table summary is in HTML examples like https://civicactions.github.io/openacr/drupal-10-16-simple.html or https://civicactions.github.io/openacr/drupal-9.html.
2. For 508 since there is no categories, I just set the column header to 'All'. I am not sure if that is valid. Do you have a suggestion for that?
